x-pack/plugins/security_solution/public/common/components/markdown_editor/plugins/index.ts

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+// TODO: This import can be removed, once https://github.com/elastic/eui/pull/7221 made its way into Kibana main
+import type { DefaultEuiMarkdownProcessingPlugins } from '@elastic/eui/src/components/markdown_editor/plugins/markdown_default_plugins/processing_plugins';
```

Comment:
Linking https://github.com/elastic/eui/pull/7221 here to keep track of it

Reply:
@janmonschke , now that the linked PR is merged. Do you want to go ahead and remove the import in this PR only?

Reply:
@logeekal Oh, did they already update EUI in Kibana?

Reply:
nope